### PR TITLE
feature : set up api service for get quiz data

### PR DESCRIPTION
**Pull request by bangmar**
done : 

- [x] yellow mark for doubt answer
- [x] set up api service and hook for take and get quiz data

### CODE REVIEW

File: apps/lxp/modules/mytraining-quiz/hooks.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { UseQueryResult, useQuery } from "@tanstack/react-query";
 import { TMyTrainingModuleResponse, params } from "../mytraining-module/type";
```
Comment (maulanasdqn) [inline apps/lxp/modules/mytraining-quiz/hooks.ts]: nama types nya jangan params atuh

TQuizParams

File: apps/lxp/modules/mytraining-quiz/section/StartQuiz/hook.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { UseQueryResult, useQuery } from "@tanstack/react-query";
+import { TGetQuizPayload, TQuizQuestionResponse } from "./types";
+import { TMetaErrorResponse } from "@mknows-frontend-services/utils";
+import { getQuizQuestion } from "./api";
+
+export const useGetQuizQuestion = (
+  payload: TGetQuizPayload,
+): UseQueryResult<TQuizQuestionResponse, TMetaErrorResponse> =>
+  useQuery({
+    queryKey: ["quiz-questions-lxp", payload.batchId, payload.quizId],
```
Comment (maulanasdqn) [inline apps/lxp/modules/mytraining-quiz/section/StartQuiz/hook.ts]: langusng aja payload gak usah di explicit lagi

File: apps/lxp/modules/mytraining-quiz/section/StartQuiz/types.d.ts [hunk 3]
```diff
@@ -1 +1,26 @@
...
+  duration: number;
+}
+
+export interface IQuestionsAnswer {
+  id: string;
+  question: string;
+  answers: Answer[];
+}
+
+export interface IAnswer {
```
Comment (maulanasdqn) [inline apps/lxp/modules/mytraining-quiz/section/StartQuiz/types.d.ts]: kalau cuma gini ngapain pake Interface

**Review by maulanasdqn — CHANGES_REQUESTED** [review]
cek cek lagi ya terutama di naming convention

File: apps/lxp/modules/mytraining-quiz/hooks.ts (hunk 1 above)
Comment (bangmar) [reply apps/lxp/modules/mytraining-quiz/hooks.ts]: bukan saya bukan saya hehe udh di fix bang

**Review by maulanasdqn — APPROVED** [review]

**Pull request merged**